src/com/griddynamics/cinema/CinemaCli.java

```diff
@@ -0,0 +1,134 @@
+package com.griddynamics.cinema;
+
+import java.util.Scanner;
+
+public class CinemaCli {
+    
+    private static final int ONE_HUNDRED_PERCENT = 100;
+    
```

Comment:
It calls magic numbers 
https://refactoring.guru/replace-magic-number-with-symbolic-constant
This discussion is interesting, please take a look at it.
https://codereview.stackexchange.com/questions/44047/100-is-a-magic-number

I think that we can avoid this constant, because looking at "100" I understand, that it is one hundred. But sometimes you can have a static program analysis tool on the project, which will fail because of "magic number"

Reply:
Oh, thank you, I will definitely take a look at that